frontend/src/components/Forms/DevPortfolioForm/DevPortfolioForm.tsx

```diff
@@ -72,9 +72,15 @@ const DevPortfolioForm: React.FC = () => {
   };
 
   const submitDevPortfolio = () => {
-    const openedEmpty = !openPRs[0] || openPRs[0].length === 0;
-    const reviewedEmpty = !reviewPRs[0] || reviewPRs[0].length === 0;
-    const otherEmpty = !otherPRs[0] || otherPRs[0].length === 0;
+    const finalPRs = openPRs.filter((pr) => pr !== '');
+    const finalReviewedPRs = reviewPRs.filter((pr) => pr !== '');
+    const finalOtherPRs = otherPRs.filter((pr) => pr !== '');
+    setOpenPRs(finalPRs);
```

Comment:
Maybe don't include these three lines changing the state? It looks a little buggy when you remove existing blank fields

Reply:
do you mean remove setOpenPRs(finalPRs);
    setReviewedPRs(finalReviewedPRs);
    setOtherPRs(finalOtherPRs); ?

Reply:
uep